google/cloud/pubsub/publisher_connection_test.cc

```diff
@@ -350,26 +355,53 @@ TEST(MakePublisherConnectionTest, TracingEnabled) {
   auto span_catcher = InstallSpanCatcher();
   auto mock = std::make_shared<pubsub_testing::MockPublisherStub>();
   Topic const topic("test-project", "test-topic");
+
+  auto mock_message_batch =
+      std::make_shared<pubsub_testing::MockMessageBatch>();
+  std::mutex mu;
+  std::condition_variable cv;
+  auto constexpr kMessageCount = 1;
+  int publish_count = 0;  // ABSL_GUARDED_BY(mu)
```

Comment:
optional: It seems marginally simpler to use a `bool published = false` when there is only one message.....

But it's fine to leave the code as is

Reply:
Done